src/components/Stap2.js

```diff
@@ -66,10 +57,13 @@ function Stap2({
 									Er is helaas geen adres bekend bij ons, beschik je zelf wel over een adres van deze instantie, vul dan
 									het onderstaande formulier in.
 								</p>
-								<SetSelectedAuthorityManual value={value} clickHandlerAuthority={clickHandlerAuthority} />
+								<SetSelectedAuthorityManual
+									value={value}
+									setAuthority={setAuthority}
```

Comment:
this part still breaks. If you enter an address here, the data changes and this input disappears